### ISSUE TITLE
clone issue

### ISSUE THREAD

**Issue by jacoweb** [magesuite/theme-generator#1]
How can this be fixed?

```
✦ ➜ /usr/local/bin/magesuite-theme-generator
Cloning CreativeShop repository to use as a base...
Command failed: git clone --branch next --single-branch 
ssh: connect to host gitlab.creativestyle.pl port 22: Operation timed out
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

**Comment by jacoweb** [comment]
can this be fixed please?

**Comment by krzksz** [comment]
I adjusted repository URL and your issue should be fixed by 046d64c08ba5d5f85729484e66637329c91df513.
Please verify and close if fixed, thanks for your patience!

**Comment by jacoweb** [comment]
Works! THank you
